### PR TITLE
Sponsor table review

### PR DESCRIPTION
**Pull request by nicpuppa**
Closes: #101

### CODE REVIEW

File: components/SponsorTable.tsx [hunk 1]
```diff
@@ -22,6 +22,9 @@ const SponsorTable = ({isVisible} : TSponsorTable) => {
                     <a href="https://www.wearedevelopers.com/" target="_blank" rel="noreferrer">
                     <Image width={150} height={150} src="/WeAreDevelopers_logo.svg" alt="WeAreDevelopers" />
                     </a>
+                </div>
+                <h3 className="sponsors_tier">Community</h3>
```
Comment (davideimola) [inline components/SponsorTable.tsx:26]:
```suggestion
                <h3 className="sponsors_tier">Media & Community</h3>
```

File: components/SponsorTable.tsx [hunk 2]
```diff
@@ -31,9 +34,6 @@ const SponsorTable = ({isVisible} : TSponsorTable) => {
                     <a href="https://pisa.dev/" target="_blank" rel="noreferrer">
                     <Image width={150} height={150} src="/pisa-dev_logo.svg" alt="PisaDev" />
                     </a>
-                    <a href="https://firenze.dev/" target="_blank" rel="noreferrer">
-                    <Image width={100} height={100} src="/firenze-dev_logo.svg" alt="Firenzedev" />
-                    </a>
```
Comment (davideimola) [inline components/SponsorTable.tsx:36]: I think we can wait a little bit before removing it definitively.

File: components/SponsorTable.tsx (hunk 1 above)
Comment (nicpuppa) [reply components/SponsorTable.tsx:26]: Don't undestand this suggestion, I separated the two partnerships

File: components/SponsorTable.tsx (hunk 2 above)
Comment (nicpuppa) [reply components/SponsorTable.tsx:36]: I'm not agree, when we know their intention we add their logo again

Comment (TheJoin95) [reply components/SponsorTable.tsx]: I agree on Nic

**Review by TheJoin95 — APPROVED** [review]
To me is ok, we need to wait a message from them as we asked them two weeks ago to let us know the way they would like to contribute to this conference project.

Until that time, we can remove the logo

**Comment by nicpuppa** [comment]
@TheJoin95 they said that they will help us (@404answernotfound correct me if I'm wrong), so we can leave their logo

**Pull request merged**